specification/ai/Azure.AI.Projects/evaluations/routes.tsp

```diff
@@ -55,6 +55,32 @@ interface Evaluations {
     Evaluation
   >;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Creates an evaluation run for agent continuous evaluation.")
+  @route("runs:agents")
+  @post
+  createAgentRun is Azure.Core.Foundations.Operation<
+    {
+      @doc("Agent evaluation with synchronous run")
+      @body
+      evaluation: AgentEvaluation;
+    },
+    Array<AgentEvaluationResult>
+  >;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations"
+  @doc("Creates an evaluation run for agent continuous evaluation.")
+  @route("runs:agents/submit")
+  @post
+  createAgentRunAsync is Azure.Core.Foundations.Operation<
```

Comment:
Questions from previous meetings: `Why are the creation APIs not returning a job status?`